File: 백준/Silver/7568. 덩치/덩치.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n;
        int k;

        n = sc.nextInt();

        int[] weight = new int[n];
        int[] height = new int[n];

        for (int i = 0; i < n; i++) {
            weight[i] = sc.nextInt();
            height[i] = sc.nextInt();
        }

        for (int i = 0; i < n; i++) {
            k = 0;

            for (int j = 0; j < n; j++) {
                if (weight[i] < weight[j] && height[i] < height[j]) {
                    k++;
                }
            }

            System.out.print(k + 1 + " ");
        }

        sc.close();
    }
}

File: 백준/Silver/7568. 덩치/덩치.cc
#include <iostream>

using namespace std;

int main() {
	int n;
	int k;

	int weight[50] = { 0, };
	int height[50] = { 0, };

	cin >> n;

	for (int i = 0; i < n; i++) {
		cin >> weight[i] >> height[i];
	}

	for (int i = 0; i < n; i++) {
		k = 0;

		for (int j = 0; j < n; j++) {
			if (weight[i] < weight[j] && height[i] < height[j]) {
				k++;
			}
		}

		cout << k + 1 << " ";
	}

	return 0;
}